projects/court-case-and-delius/build.gradle.kts

```diff
@@ -13,13 +13,15 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
     implementation("org.springframework.boot:spring-boot-starter-security")
     implementation("org.springframework.boot:spring-boot-starter-validation")
+    implementation("org.springframework.boot:spring-boot-starter-data-ldap")
```

Comment:
You'll need to add references to the LDAP creds in the Helm values.yaml file for this too.

Reply:
done